setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = django-on-demand
-version = 0.1
+version = 0.2
```

Comment:
Is a must to bump this version?

Reply:
this was just for testing, in order to upload a new version to the package manager I need to bump the version